app/templates/account.hbs

```diff
@@ -3,19 +3,17 @@
 <header>
   {{#unless config.enterprise}}
     {{#if config.billingEndpoint}}
-      <div class="cta-btn">
-        {{#if subscribeButtonInfo.subscribed}}
-          <a class="button" href={{billingUrl}}>Subscription active!</a>
+      {{#if subscribeButtonInfo.subscribed}}
+        <p class="notice-banner--green">This account has an active subscription. <a  href={{billingUrl}} title="Go to Travis CI Billing" class="right">Manage your subscription!</a></p>
```

Comment:
It might be nice to extract a component here. That would allow us to get rid of some of the repeated markup.

Something like:

```hbs
{{subscription-status-banner 
  color="green" 
  messageText="This account has an active subscription"
  billingLinkText="Manage your subscription!"
}}
```